snapshot/snapshot.go

```diff
@@ -282,6 +284,15 @@ func (o *snapshotter) Mounts(ctx context.Context, key string) ([]mount.Mount, er
 		return nil, errors.Wrapf(err, "get snapshot %s info", key)
 	}
 
+	metricBeginTime := time.Now()
+	defer func(id *string) {
+		metricTime := time.Since(metricBeginTime)
+
+		if err := exporter.ExportMountTimeMetric(*id, metricTime.Seconds()); err != nil {
```

Comment:
Seconds are too rough, can we use milliseconds or use float values

Reply:
It seems that the `metricTime.Seconds()` is already a float value for seconds.

```go
// Seconds returns the duration as a floating point number of seconds.
func (d Duration) Seconds() float64 {
	sec := d / Second
	nsec := d % Second
	return float64(sec) + float64(nsec)/1e9
}
```